### PR TITLE
Chinese Translation update

### PR DESCRIPTION
**Pull request by WWWWWWWWWWWWWWWWWWWWang**
Three more translation files and I updated the main translation file this time.

### CODE REVIEW

File: BondageClub/Scripts/Translation.js [hunk 1]
```diff
@@ -373,6 +373,7 @@ var TranslationDictionary = [
 			"Screens/Cutscene/PlayerCollaring/Text_PlayerCollaring_CN.txt"
 			"Screens/Cutscene/PlayerMistress/Text_PlayerMistress_CN.txt"
 			"Screens/Cutscene/SarahIntro/Text_SarahIntro_CN.txt"
+			"Screens/Cutscene/AsylumBedroom/Text_AsylumBedroom_CN.txt"
```
Comment (remiliacn) [inline BondageClub/Scripts/Translation.js]: `Blocking Typo`

Shouldn't this be `[Screens/Cutscene/SarahIntro/Text_SarahIntro_CN.txt](https://github.com/Ben987/Bondage-College/pull/3307/commits/84607926cb3e7791e2ea6314252f8d2bc22e64ad#diff-8f6e938610c128e3daf945db37e4f4e03e6932dedb7faa2043afc846c52f1cf8)`?

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+The social room is a good place to start.  Or get to the beds to relax.  Now get going girl.
+可以先从社交室开始？或者回房间小睡一会也行。总之，行动起来吧！
+###_PLAYER
+You're locking the door
+你把门关上了
+###_NPC
+Don't be afraid, it's for your own good.  Now get going girl, you can explore the asylum freely.
+别害怕孩子，这是对你好。现在行动起来吧，你可以在收容所内任意走动。
+By wearing this.  (She straps a straitjacket on you.)  Patients like you must be kept under control.  Now carry on, you know the place.
+想你这种病人必须被好好的控制住（她给你穿上了拘束服）行了，你知道接下来该去哪儿。
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:187]: > 想你这种病人

想 -> 像

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 3]
```diff
@@ -0,0 +1,578 @@
...
+What is going on Miss
+发生什么事了,小姐？
+What the fuck
+什么鬼？
+What is that
+那是什么？
+###_NPC
+GGTS is a computer intelligence installed in the asylum to train obedience.  You've been selected to participate.
+GGTS是一个人工智能
+###_PLAYER
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:500]: 以下的部分可以翻译，不涉及到GGTS的核心玩法

We could translate the part below, those are normal conversation that should not affect gameplay.

File: BondageClub/Screens/Cutscene/PlayerMistress/Text_PlayerMistress_CN.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Mistresses from the Bondage Club gather to meet you and introduce themselves.
+晋升派对前，俱乐部的管理员们与你有了一次见面，并且向你介绍了自己。
+They congratulate you on your promotion and talk about future club events.
+他们对你的晋升表示了祝贺，并讨论了未来俱乐部的活动方向。
+You're provided with the official Bondage Club Mistress padlocks and keys.
+接着，你收到了俱乐部管理员专用的挂锁以及钥匙。
+After a while, they bring in the bound maids to serve drinks and refreshments.
+过了一会儿，其他管理员带来了一位被捆绑的女仆来为你们提供饮料和茶点。
+You meet many new Dominant friends who now address you as "Mistress".
+作为新的管理员，你在派对上遇到了很多新的DOM朋友。
```
[the remark this answers is not in the document]
Comment (WWWWWWWWWWWWWWWWWWWWang) [reply BondageClub/Screens/Cutscene/PlayerMistress/Text_PlayerMistress_CN.txt:10]: 好的

File: BondageClub/Scripts/Translation.js (hunk 1 above)
Comment (WWWWWWWWWWWWWWWWWWWWang) [reply BondageClub/Scripts/Translation.js]: I will fix this now. Thanks for reviewing ( ´ ▽ ` )b

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt (hunk 3 above)
Comment (WWWWWWWWWWWWWWWWWWWWang) [reply BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:500]: 已解决，昨天误操作把翻译完的删了有问题留下来了（悲

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 5]
```diff
@@ -412,7 +412,7 @@ Hello.  I'm DialogPlayerName.
 It's you!  You're the patient that escaped.  Get her girls!  (Lots of nurses come to help her.)
 是你！那个逃跑的病人！姐妹们，抓住她！（很多护士都跑出来准备帮她）
 ###_PLAYER
-Know me? It's my first visit here.
+Know me?  It's my first visit here.
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:415]: `nit - non-blocking`

Double spaces?

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 6]
```diff
@@ -440,19 +440,19 @@ What's going on?
...
 That's unfair!
-那不公平！
+不带这么玩的啊！
 ###_NPC
 It's very fair.  Welcome back patient, I'm sure your forced day here will teach you a lesson.
-这很公平。欢迎回来，病人。你必须在这里呆上一整天来好好矫正你的行为
+就这这么玩的~欢迎回来，病人。你必须在这里呆上一整天来好好矫正你的行为
 ###_PLAYER
-One full day? Are you insane?
+One full day?  Are you insane?
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:455]: `nit - non-blocking`

double spaces?

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 7]
```diff
@@ -472,107 +472,107 @@ Very well.  (She smiles.)  Now carry on, I have work to do.
...
+可以的，无人机仍旧可以使用收容所的所有功能
 ###_PLAYER
-Can you let me out
-
+Can you let me out?
+你可以让我出去吗？
 ###_NPC
 (She nods.)  Drones can leave the asylum if they spend ten GGTS minutes.
-
+（她点了点头）无人机想出去的话，你得话10个GGTS时间
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:543]: > 你得话10个GGTS时间

花

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 8]
```diff
@@ -472,107 +472,107 @@ Very well.  (She smiles.)  Now carry on, I have work to do.
...
-
+（她看了看你，叹了口气）小无人机啊，这规则不是我定的，不要给我们制造麻烦。
 ###_PLAYER
 I'll force my way out.
-
+我会用些强硬的方法出去的
 ###_NPC
 (She smirks.)  Good luck, the doors are bolted with magnets.
-
+（她笑了起来）那祝你好运。顺带一提，这里的们都是用磁关的
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:569]: > 这里的们都是用磁关的

门

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt (hunk 5 above)
Comment (WWWWWWWWWWWWWWWWWWWWang) [reply BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:415]: This is the original text, so I think I shouldn't change this.

File: BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt [hunk 9]
```diff
@@ -0,0 +1,578 @@
...
+No, as a club slave you have work to do.  Get going girl.
+不，你还有作为俱乐部奴隶的工作要做，快离开吧。
+###_PLAYER
+(Leave her.)
+（离开她）
+###_NPC
+Greetings doctor DialogPlayerName.  It's always a pleasure to see you.  Are you working today?
+你好啊DialogPlayerName医生，很高兴见到你！请问你今天要工作吗？
+Welcome to the asylum nurse DialogPlayerName.  Our patients are expecting you.  Are you here to work?
+欢迎来到疗养院，DialogPlayerName护士。病人们都很期待你能来，请问你今天要工作吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:23]: `suggestion - blocking`

asylum这个词的翻译可以统一一下嘛，`疗养院` or `收容所`

问题举例：

https://github.com/Ben987/Bondage-College/blob/e434862b429b5a53d331ceba119078a81b6b0962/BondageClub/Screens/Room/AsylumBedroom/Text_AsylumBedroom_CN.txt#L5-L6

https://github.com/Ben987/Bondage-College/blob/e434862b429b5a53d331ceba119078a81b6b0962/BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt#L22-L23

Comment (WWWWWWWWWWWWWWWWWWWWang) [reply BondageClub/Screens/Room/AsylumEntrance/Dialog_NPC_AsylumEntrance_Nurse_CN.txt:23]: 抱歉，这就去改

File: BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt [hunk 10]
```diff
@@ -0,0 +1,578 @@
...
+No, as a club slave you have work to do.  Get going girl.
+不，你还有作为俱乐部奴隶的工作要做，快离开吧。
+###_PLAYER
+(Leave her.)
+（离开她）
+###_NPC
+Greetings doctor DialogPlayerName.  It's always a pleasure to see you.  Are you working today?
+你好啊 DialogPlayerName 医生， 很高兴见到你！请问你今天要工作吗？ 
+Welcome to the asylum nurse DialogPlayerName.  Our patients are expecting you.  Are you here to work?
+欢迎来到疗养院，DialogPlayerName 护士。病人们都很期待你能来，请问你今天要工作吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt:23]: `suggestion - blocking`

翻译不统一的问题~

File: BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt [hunk 11]
```diff
@@ -0,0 +1,578 @@
...
+Hello again DialogPlayerName.  Are you here to check in?
+你好DialogPlayerName，你是来报到的吗？
+Welcome to the asylum.  Do you need information?
+欢迎来到收容所，你想了解什么吗？
+###_PLAYER
+I want to learn about the asylum.
+我想了解一些关于收容所的事
+###_NPC
+In the asylum, you can work as a nurse or commit yourself as a patient.  What would you like to know?
+在庇护所里，你可以作为一名护士工作，也可以作为一名病人。你想了解些什么吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt:39]: `suggestion - blocking`

翻译不统一的问题~

File: BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt [hunk 12]
```diff
@@ -0,0 +1,578 @@
...
+并不（叹气）。收容所有了新规定，护士现在得在上班时戴项圈了。
+###_PLAYER
+Can I get my own nurse uniform?
+我可以要一套护士制服吗？
+###_NPC
+Of course.  Here you go nurse DialogPlayerName.  Wear it with pride.  (She gives you a nurse uniform.)
+当然了（她帮你拿了一套制服）。拿好了，DialogPlayerName，骄傲的穿上它吧！
+###_PLAYER
+Can I get my own set of Asylum restraints?
+我可以有一套专属的庇护所拘束套组吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt:70]: `suggestion - blocking`

翻译不统一的问题~

File: BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt [hunk 13]
```diff
@@ -0,0 +1,578 @@
...
+（她看着你并点了点头）你好啊，附属机（Drone）
+###_PLAYER
+Why are you calling me drone?
+你为什么要叫我Drone？
+###_NPC
+Look at you girl.  You're a drone, a total GGTS slave.
+看看你自己吧，你就是一个附属机（Drone），一个完全的GGTS的奴隶
+###_PLAYER
+Can I still use the asylum?
+我还能使用庇护所吗？
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt:534]: `suggestion - blocking`

翻译不统一的问题~

File: BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+Your Profile
+你的档案
+Asylum Chat Rooms
+收容所聊天室
+Your Bedroom
+你的卧室
+Meeting Space
+会议室
+Therapy Room
+治疗室
```
Comment (remiliacn) [inline BondageClub/Screens/Room/AsylumEntrance/Text_AsylumEntrance_CN.txt:12]: `Suggestion - non-blocking`

这个翻译成理疗室会不会更好一点？只是建议~

**Review by remiliacn — COMMENTED** [review]
感觉应该没问题了~

LGTM.

**Review by remiliacn — APPROVED** [review]

**Comment by Ben987** [comment]
Very nice!  Thanks a lot you two.  I'm sure that Chinese players will appreciate :)

**Pull request merged**